hs_core/tasks.py

```diff
@@ -539,3 +539,54 @@ def update_quota_usage_task(username):
     uq.update_used_value(used_val)
 
     return True
+
+
+@shared_task
+def update_web_services(services_url, api_token, timeout, publish_urls, res_id):
+    """Update web services hosted by GeoServer and HydroServer.
+
+    This function sends a resource id to the HydroShare web services manager
+    application, which will check the current status of the resource and register
+    or unregister services hosted by GeoServer and HydroServer.
+    The HydroShare web services manager will return a list of endpoint URLs
+    for both the resource and individual aggregations. If publish_urls is set to
+    True, these endpoints will be added to the extra metadata fields of the
+    resource and aggregations.
+    """
+    session = requests.Session()
+    session.headers.update(
+        {"Authorization": " ".join(("Token", str(api_token)))}
+    )
+
+    rest_url = str(services_url) + "/" + str(res_id) + "/"
+
+    try:
+        response = session.post(rest_url, timeout=timeout)
+
+        if publish_urls is True and response.status_code == 201:

```

Comment:
@kjlippold This can be simplified with just ```if publishe_urls```. Also, it'd be good to use status constant such as https://github.com/hydroshare/hydroshare/blob/94528abd61b76dd1f04b40989b3a9c3ff2dc901a/hs_core/views/resource_folder_hierarchy.py#L38 rather than a number.

Reply:
Alright, I've simplified the if statement and used drf status codes in the statement.